build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -942,6 +945,17 @@ message ExecutedActionMetadata {
   // When the worker completed executing the action command.
   google.protobuf.Timestamp execution_completed_timestamp = 8;
 
+  // The amount of time the worker spent executing the action command,
+  // not including overheads before or after execution such as
+  // marshalling inputs/outputs.
+  //
+  // The server SHOULD avoid including time spent the client doesn't
+  // have control over, and MAY reduce the execution duration to account
+  // for delays that occur during execution itself (e.g., lazily loading
+  // data from the Content Addressable Storage, live migration of
+  // virtual machines, emulation overhead).
+  google.protobuf.Duration execution_duration = 12;
```

Comment:
Do you think we need "new in v2.3" to indicate this is a new metadata field? Or do we think that all metadata being optional suffices to save us here - no caller should ever depend on this being populated, so when it was introduced isn't semantically meaningful?

(Having typed this, we're probably fine to leave it off. But I'll still send this comment to get a second opinion :))

Reply:
Seems like a good idea to do that. Added!